src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -100,6 +100,75 @@ public void start() {
       addBombardmentSources();
       m_needToAddBombardmentSources = false;
     }
+    
+    Territory lastAmphib = null, lastNonAmphib = null, t;
+    IBattle battle;
+    Iterator<Territory> battleTerritories = m_battleTracker.getPendingBattleSites(true).iterator(); // get bombing/air raids
+    if( battleTerritories.hasNext() ) {
+      t = battleTerritories.next();
+      if( battleTerritories.hasNext() ) return; // If more than one air/boming raid do nothing here
+      battle = m_battleTracker.getPendingBattle(t, true, null);
+      if( battle == null ) {
+        System.out.println("Air/Bombing Raid gone missing in BattleDelegate");
```

Comment:
Hmm, it should never happen. How should you flag such a thing - throw an exception then?

Probably could use a for/foreach but why? It's easier the way I've done it. We really only care about the first answer. If there are multiple answers, we do nothing. With a for/foreach, you'd have to count the number of times you go around the loop.

I'm not much of a Java programmer so it might not look like normal Java code.